po/pt_BR.po

```diff
@@ -0,0 +1,216 @@
+# Portuguese translations for com.github.tkashkin.gamehub package.
+# Copyright (C) 2018 THE com.github.tkashkin.gamehub'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the com.github.tkashkin.gamehub package.
+# Automatically generated, 2018.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: com.github.tkashkin.gamehub\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-07-13 16:18+0300\n"
+"PO-Revision-Date: 2018-07-13 16:18+0300\n"
+"Last-Translator: Automatically generated\n"
+"Language-Team: none\n"
+"Language: pt_BR\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#: data/com.github.tkashkin.gamehub.appdata.xml.in:8
+#: data/com.github.tkashkin.gamehub.desktop.in:3
+#: data/com.github.tkashkin.gamehub.desktop.in:4
+msgid "GameHub"
+msgstr "GameHub"
+
+#: data/com.github.tkashkin.gamehub.appdata.xml.in:9
+#: data/com.github.tkashkin.gamehub.desktop.in:5
+#: src/ui/views/WelcomeView.vala:38
+msgid "All your games in one place"
+msgstr "Todos os seus jogos em um só lugar"
+
+#: data/com.github.tkashkin.gamehub.appdata.xml.in:12
+msgid "Manage your Steam, GOG and Humble Bundle games in one place."
+msgstr "Gerencie seus jogos Steam, GOG e Humble Bundle em um só lugar."
+
+#: src/data/Game.vala:73
+msgid "Installed"
+msgstr "Instalado"
+
+#: src/data/Game.vala:74
+msgid "Installing"
+msgstr "Instalando"
+
+#: src/data/Game.vala:75
+msgid "Download started"
+msgstr "Começou a baixar"
+
+#: src/data/Game.vala:76
+msgid "Download finished"
+msgstr "Jogo baixado com sucesso"
+
+#: src/data/Game.vala:79
+#, c-format
+msgid "Downloading: %d%% (%s / %s)"
+msgstr "Baixando: %d%% (%s / %s)"
+
+#: src/data/Game.vala:81
+msgid "Not installed"
+msgstr "Não instalado"
+
+#: src/data/sources/steam/Steam.vala:17
+msgid "Your SteamID will be read from Steam configuration file"
+msgstr "Seu SteamID será lido a partir do arquivo de configuração do Steam"
+
+#: src/data/sources/steam/Steam.vala:20
+msgid ""
+"Steam config file not found.\n"
+"Login into your account in Steam client and return to GameHub"
+msgstr ""
+"Steam config file not found.\n"
+"Entre na sua conta no cliente Steam e retorne ao GameHub"
```

Comment:
@LeandroStanger Please translate this line.

Reply:
@LeandroStanger it's multiline string and should be translated like [this](https://github.com/tkashkin/GameHub/blob/master/po/ru.po#L66-L72). Syntax:
```
msgstr ""
msgstr "line1\n"
msgstr "line2"
```
All lines except last should end with \n